docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,62 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## SWAP
+
+Endpoints to make SWAP transactions
+
+### List of tokens available for SWAP
+
+`GET /v1/swap/tokens`
+
+#### Path parameters
+
+* `projectId` - is the project identifier
+* `chainId` - filter by chain. CAIP-10 format.
+
+#### Success response body:
+
+* `tokens` - list of objects which define available tokens:
+    * `name` - token name.
+    * `symbol` - token symbol.
+    * `address` - contract address of token.
+    * `chainId` - ID of chain in CAIP-10 format.
+    * `decimals` - decimals for amount supported by a given token.
+    * `iconUrl` - URL of the token icon.
+
+#### Response error codes:
+
+* `400 Bad Request` - some parameters in request body were missed or wrong.
+* `401 Unauthorized` - projectID verification error.
+
+### Requesting SWAP quotes
+
+`GET /v1/swap/quotes`
+
+#### Path parameters
+
+* `projectId` - is the project identifier.
+* `amount` - amount of tokens to be swapped.
+* `fromChainId` - ID of sending chain.
+* `fromAddress` - source address.
+* `toChainId` - ID of destination chain.
+* `toAddress` - destination address.
+
+#### Success response body:
+
+* `quotes` - list of objects which define available swap quotes
+    * `id` - quote ID to be used for swap request.
+    * `fromAmount` - amount of token being sent on sending chain.
+    * `fromChainId` - ID of sending chain.
+    * `fromToken` - object metadata of sending token same as from `List of tokens available for SWAP`.
+    * `fromAddress` - source address.
+    * `toAmount` - amount of token to be received on the destination chain.
```

Comment:
any format required for this? eg. `1`. `1.1` `1.100000000` are they all valid? what about something like `000.0` that'd break right?

Reply:
This should be a string and it's related to the decimals for the token.e.g. 100000000 for 100 USDC because USDC is 6 decimals. I should add this explanation to the description.